Lib/test/test_asyncio/test_sendfile.py

```diff
@@ -87,9 +87,13 @@ async def wait_closed(self):
 
 class SendfileBase:
 
-      # 128 KiB plus small unaligned to buffer chunk
-    DATA = b"SendfileBaseData" * (1024 * 8 + 1)
-
+    # 256 KiB plus small unaligned to buffer chunk
+    # Newer versions of Windows seems to have increased its internal 
+    # buffer and tries to send as much of the data as it can as it 
+    # has some form of buffering for this which is less than 256KiB
+    # on newer server versions and Windows 11.
+    # So DATA should be larger than 256 KiB to make this test reliable.
+    DATA = b"x" * (1024 * 256 + 1)
```

Comment:
I keep coming back to this. I don't think it's worth changing anything (too much churn), but I would still like to point out that the original code had 16 bytes at the end of the 128 kb; the new code has only one extra byte after the 256 kb.

Also, this change affects almost all tests in this file, rather than just the one flaky test.

Finally, the use of "SendfileBaseData" as the dummy data seemed to be intentional to help debugging the test (e.g. test_sock_sendfile_mix_with_regular_send).

Just making sure you were aware of all these. :-)

Reply:
I didn't read the original PR. I only helped to backport the change because IMO it's better to run a test than skipping it in 3.9 and 3.10 branch.

@kumaraditya303: Do you want to write a PR to make changes suggested by @gvanrossum?